src/sage/schemes/curves/zariski_vankampen.py

```diff
@@ -22,10 +23,12 @@
 
 EXAMPLES::
 
-    sage: from sage.schemes.curves.zariski_vankampen import fundamental_group # optional - sirocco
-    sage: R.<x,y> = QQ[]
+    sage: from sage.schemes.curves.zariski_vankampen import fundamental_group, braid_monodromy  # optional - sirocco
+    sage: R.<x, y> = QQ[]
     sage: f = y^3 + x^3 - 1
-    sage: fundamental_group(f) # optional - sirocco
+    sage: braid_monodromy(f)                        # optional - sirocco
+    ([s1*s0, s1*s0, s1*s0], {0: 0, 1: 0, 2: 0})
+    sage: fundamental_group(f)                      # optional - sirocco
     Finitely presented group < x0 |  >
```

Comment:
block-scoped?

Reply:
Yes; most of the doctests having several `# optional - sirocco` are only meaningful with sirocco and I block-scoped most of them.

Reply:
Thanks.